### ISSUE TITLE
Counter-Strike 2

### ISSUE THREAD

**Issue by AndreDavis909** [worse-666/csgo_external_ahk_hack#56]
Hello! please make the same scripts for Counter-Strike 2.

**Comment by AndreDavis909** [comment]
Is it possible to have a script for highlighting enemy players?

**Comment by ilyadex** [comment]
I don’t think that the anti-cheat will work there; they updated and ahk detects many times better and the cheat itself has been detected and the entire cheat needs to be rewritten

**Comment by AndreDavis909** [comment]
> I don’t think that the anti-cheat will work there; they updated and ahk detects many times better and the cheat itself has been detected and the entire cheat needs to be rewritten

I am needed for training with bots)

**Comment by AndreDavis909** [comment]
> I don’t think that the anti-cheat will work there; they updated and ahk detects many times better and the cheat itself has been detected and the entire cheat needs to be rewritten

Are you the author of this script?
